articles/server-platforms/nancyfx/01-login.md

```diff
@@ -31,7 +31,7 @@ The `RedirectOnLoginFailed` specifies the view that should be shown to an authen
 
 The `CookieName` allows you to set the name of the cookie that will be used to save the User information.
 
-The `UserIdentifier` lets you set an identifier for the user. Currently, here are the fields that are available:
+The `UserIdentifier` let you set an identifier for the user. Currently, here are the fields that are available:
```

Comment:
@camimonse This doesn't look right. `Lets` is used in the third-person singular present tense, and `The UserIdentifier` is an `it`. If we used `allow` instead of `let`, we would say `It allows you to set an identifier for the user` (where It=The UserIdentifier).